src/Education/Commands/RuleHelpUserControl.xaml

```diff
@@ -2,111 +2,142 @@
+                <Setter Property="Margin" Value="0,10,0,5" />
+            </Style>
+            <Style x:Key="Table" TargetType="Table">
+                <Setter Property="BorderBrush" Value="{DynamicResource {x:Static vsshell:VsBrushes.PanelBorderKey}}"/>
+                <Setter Property="BorderThickness" Value="2" />
+                <Setter Property="Margin" Value="2" />
+                <Setter Property="Padding" Value="1" />
+            </Style>
+            <Style x:Key="TableHeaderRowGroup" TargetType="TableRowGroup">
+                <Setter Property="FontWeight" Value="{DynamicResource {x:Static vsshell:VsFonts.EnvironmentBoldFontWeightKey}}"/>
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.InactiveCaptionKey}}" />
+            </Style>
+            <Style x:Key="TableHeaderCell" TargetType="TableCell">
+                <Setter Property="BorderBrush" Value="{DynamicResource {x:Static vsshell:VsBrushes.PanelBorderKey}}"/>
+                <Setter Property="BorderThickness" Value="0,0,0,1"/>
+                <Setter Property="Padding" Value="8,5,5,8" />
+            </Style>
+            <Style x:Key="TableBodyCell" TargetType="TableCell">
+                <Setter Property="Padding" Value="8,5,5,8" />
+            </Style>
+            <Style x:Key="TableBodyCellAlternateRow" TargetType="TableCell">
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoBackgroundKey}}" />
+                <Setter Property="Padding" Value="8,5,5,8" />
+            </Style>
+            <Style x:Key="UnorderedList" TargetType="List">
+                <Setter Property="MarkerStyle" Value="Circle" />
+                <Setter Property="Margin" Value="2" />
+            </Style>
+            <Style x:Key="OrderedList" TargetType="List">
+                <Setter Property="MarkerStyle" Value="Decimal" />
+                <Setter Property="Margin" Value="2" />
+            </Style>
+            <Style x:Key="Code_Span" TargetType="Span">
+                <Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoTextKey}}"/>
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoBackgroundKey}}" />
+                <Setter Property="FontFamily" Value="Courier New" />
+                <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment90PercentFontSizeKey}}"/>
+            </Style>
+        </ResourceDictionary>
+    </UserControl.Resources>
 
     <FlowDocumentScrollViewer x:Name="docViewer" Hyperlink.RequestNavigate="HandleRequestNavigate">
-        <FlowDocument >
+        <FlowDocument>
+            
+            <!-- Design-time only control to check the embedding and rendering of the images.
+                 The d: prefix means this won't appear in the runtime build. -->
+            <d:BlockUIContainer>
```

Comment:
This section is new, but only appears in the IDE because of the `d"` prefix. It allows us to check that the converted SVG images are imported correctly and see what they look like at design time.

#### View in the IDE
![image](https://user-images.githubusercontent.com/10757559/219885224-32dce4f6-3cd4-4320-b8ff-04029afbd1e0.png)

#### Appearance at runtime
![image](https://user-images.githubusercontent.com/10757559/219885220-cf177437-2f0e-49bc-9e08-2f8c527b6a93.png)

Reply:
The new images scale well:

![image](https://user-images.githubusercontent.com/10757559/219885461-ac52297a-188f-4c74-b83d-0c7ed24dd62f.png)